.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Testing
+
+**You should manually test your PR before merging.**
+
+Example steps to test in Integration:
```

Comment:
Are we asking people to test in all environments? or just integration? or any environment they want?

Reply:
It's a good question. I didn't want to be too prescriptive, as _theoretically_ people should follow our normal deployment process and therefore do a branch deploy of their change in each environment (since a merge "deploys" to all simultaneously). Do you think people won't exercise common sense here?

Reply:
Hmm, yeah its tricky. I guess we get into the grey area of common sense - I'd probably hesitate to test in production. I think if people **_should_** be testing in every environment, then we should be explicit and let people use common sense if they deviate from that?

Reply:
@theseanything I've updated this as-per our discussion. Can you take another look?